ern-local-cli/src/lib/publication.js

```diff
@@ -546,21 +547,23 @@ async function getCodePushAppName (
   return result
 }
 
-export function getCodePushAccessKey () {
+export function getCodePushInitConfig (): CodePushInitConfig {
   const codePushConfigFilePath = path.join(process.env.LOCALAPPDATA || process.env.HOME || '', '.code-push.config')
+  var codePushInitConfig: CodePushInitConfig
   if (fs.existsSync(codePushConfigFilePath)) {
-    return JSON.parse(fs.readFileSync(codePushConfigFilePath, 'utf-8')).accessKey
+    codePushInitConfig = JSON.parse(fs.readFileSync(codePushConfigFilePath, 'utf-8'))
   } else {
-    return config.getValue('codePushAccessKey')
+    codePushInitConfig = {'accessKey': config.getValue('codePushAccessKey')}
```

Comment:
could you also add headers/custom url/proxy here.
you can just assume the config properties to be prefixed by `codePush` as we did for access key. we will document that later that's ok, but at least the code is there. 
this can be needed in CI environments if we don't want to install the Code Push CLI on the box (and therefore don't end up having a `.code-push.config` on the box)

Reply:
+1